backend/src/main/java/com/google/models/DataInfoUser.java

```diff
@@ -84,8 +84,8 @@ public boolean equals(Object o) {
 
   public static DataInfoUser createFromEntity(Entity entity) {
     return DataInfoUser.of(
-      (String) entity.getProperty(Constant.CONFIG_METRIC_PROPERTY),
-      (String) entity.getProperty(Constant.CONFIG_DIMENSION_PROPERTY),
+      (String) entity.getProperty(Constant.CONFIG_RMETRIC_PROPERTY),
```

Comment:
probably missed this in an earlier RMETRIC is less readable than `RELATED_METRIC`.

Also as an fyi, sometimes its possible to have nested Constants
ie Constant.Config.RELATED_METRIC_PROPERTY

to gropu different constants together.

Reply:
yep, i will rename it and try to see if nested Constants work!

Reply:
I think I will move the Constants once Leo merges his code because then there will be a `Configuration` class and I can move these configuration related constants there.